### PR TITLE
Support additional properties

### PR DESCRIPTION
**Pull request by fredr**
Adds support for linking with pull request via pull request number, and for adding meta_data to the build

### CODE REVIEW

**Comment by fredr** [comment]
Any chance this could get a review? maybe @lizrabuya  or @EnricoMi?
Sorry for pinging directly, but I'm not sure how to progress with this.

File: tests/entrypoint.bats
Comment (lizrabuya) [inline tests/entrypoint.bats:1]: LGTM, but can you update the tests with valid meta-data parameters 

`e.g. @test "Creates a build with meta-data vars`

**Review by lizrabuya — CHANGES_REQUESTED** [review]
Suggested for additional test case

File: tests/entrypoint.bats
Comment (fredr) [reply tests/entrypoint.bats:1]: For sure, done! Sorry for not adding that directly. I also added a test for the pull request id

**Review by lizrabuya — APPROVED** [review]
Looks great!

**Pull request merged**